docs/notebooks/LORIS_API_Part2_Python-script.ipynb

```diff
@@ -0,0 +1,2159 @@
+    "id": "m7opbczp372G"
+   },
+   "source": [
+    "## About the data in demo.loris.ca\n",
+    "The demo instance of LORIS contains the raisinbread dataset, which is used \n",
+    "by LORIS developers. For more informations about raisinbread, click [here](https://github.com/aces/Loris/tree/main/raisinbread)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "jLy1LeeyZ_HE"
+   },
+   "source": [
+    "## Exercise 1. Login\n",
+    "\n",
+    "This is a POST request to the `/login` endpoint that requires 2 parameters: `username` and `password`  \n",
+    "The expected response is a json string that contains a token property.  \n",
+    "\n",
+    "https://github.com/aces/Loris/blob/main/modules/api/docs/LorisRESTAPI_v0.0.3.md"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 106
+    },
+    "colab_type": "code",
+    "id": "VeCuzjwbZ_HG",
+    "outputId": "3c13b4b1-4de5-452e-ff2c-98c26a312bf4",
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "payload = {\n",
+    "    'username': input('username: '), \n",
+    "    'password': getpass.getpass('password: ')\n",
+    "}\n",
+    "\n",
+    "response = requests.post(\n",
+    "    url = '/'.join([baseurl, 'login']),\n",
+    "    json = payload,\n",
+    "    verify = False\n",
+    ")\n",
+    "\n",
+    "text = response.content.decode('ascii')\n",
+    "\n",
+    "data = json.loads(text)\n",
+    "\n",
+    "prettyPrint(data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "5D9tnYTTZ_HM"
+   },
+   "source": [
+    "*Store the token in a variable for later*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "NGatwtA2Z_HN",
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "token = data['token']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "S6G0Qn6uSyBn"
+   },
+   "source": [
+    "The requests thoughout this tutorial are very similar, only the URL changes. We will define 5 functions that will make the tutorial easier to read.\n",
+    "\n",
+    "For more information on HTTP requests and their differences, you can refer to https://github.com/aces/Loris/blob/main/modules/api/docs/LorisRESTAPI_v0.0.3.md#10-overview"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "unsItgX_SxPF"
+   },
+   "outputs": [],
+   "source": [
+    "def GET(url):\n",
+    "    \"\"\"\n",
+    "    Function to send an HTTP GET request\n",
+    "\n",
+    "    Parameters: baseurl (URL of the api, e.g. https://demo.loris.ca/api/v0.0.3)\n",
+    "\n",
+    "    Returns: projects (Information on all projects)\n",
```

Comment:
This is not accurate

Reply:
I changed it, but it looks like Github doesn't track Jupyter notebooks very well...